modules/build.gradle

```diff
@@ -290,6 +290,10 @@ gradle.beforeProject {
 
 					systemProperty "gradle.distribution", uri(gradleDistributionFile)
 					systemProperty "maven.distribution.file.name", configurations.mavenDistribution.singleFile
+
+					validateTLD {
```

Comment:
Should this be outside of the `test {}` closure?

Reply:
I'm not sure if we need to validateTLDs outside of the test task, like in `jar` or `install`.